Dmf/Modules.Library/Dmf_String.h

```diff
@@ -20,6 +20,40 @@ Module Name:
 
 #pragma once
 
+#if defined(DMF_USER_MODE)
+
+__forceinline
+NTSTATUS

```

Comment:
I would recommend adding _Must_inspect_result_ here

Reply:
Same comment for the other functions returning NTSTATUS codes.

Reply:
I will do that next update.